### PR TITLE
feat: Update clients

### PR DESCRIPTION
**Pull request by Wolmin**
https://app.clickup.com/t/2645698/DEV-8040

### CODE REVIEW

File: flags/flags.go [hunk 1]
```diff
@@ -189,20 +189,20 @@ var (
...
 			Name:  GethTagFlag,
 			Usage: "A tag of geth you would like to run",
-			Value: "1.12.2",
+			Value: "1.13.1",
 		},
 		&cli.StringFlag{
 			Name:  GethCommitHashFlag,
 			Usage: "A hash of commit that is bound to given release tag",
-			Value: "bed84606",
+			Value: "3f40e65c",
```
Comment (Hugoo) [inline flags/flags.go:197]: https://github.com/ethereum/go-ethereum/releases/tag/v1.13.1

Ok

**Review by Hugoo — APPROVED** [review]

**Comment by mxmar** [comment]
@Wolmin Please verify:
- https://github.com/lukso-network/tools-lukso-cli/blob/main/README.md
- https://github.com/lukso-network/tools-lukso-cli/blob/main/CONTRIBUTING.md
- https://github.com/lukso-network/tools-lukso-cli/blob/main/CHANGELOG.md

File: README.md [hunk 2]
```diff
@@ -20,10 +20,10 @@ The LUKSO CLI is able to install multiple clients for running the node.
...
 | Client     | Version | Release                                                      |
 |------------|---------|--------------------------------------------------------------|
-| Geth       | v1.12.2 | https://github.com/ethereum/go-ethereum/releases/tag/v1.12.2 |
-| Erigon     | v2.48.1 | https://github.com/ledgerwatch/erigon/releases/tag/v2.48.1   |
+| Geth       | v1.13.1 | https://github.com/ethereum/go-ethereum/releases/tag/v1.13.1 |
+| Erigon     | v2.49.2 | https://github.com/ledgerwatch/erigon/releases/tag/v2.49.2   |
 | Prysm      | v4.0.8  | https://github.com/prysmaticlabs/prysm/releases/tag/v4.0.8   |
-| Lighthouse | v4.3.0  | https://github.com/sigp/lighthouse/releases/tag/v4.3.0       |
+| Lighthouse | v4.4.1  | https://github.com/sigp/lighthouse/releases/tag/v4.4.1       |
 | Teku       | v23.8.0 | https://github.com/Consensys/teku/releases/tag/23.8.0        |
```
Comment (mxmar) [inline README.md]: I forgot to mention `Teku`. I think that we should update to `23.9.0`: https://github.com/Consensys/teku/releases/tag/23.9.0

**Review by mxmar — CHANGES_REQUESTED** [review]
Let's update `Teku` to https://github.com/Consensys/teku/releases/tag/23.9.0 (and make tests before).

**Review by mxmar — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Wolmin) [reply README.md]: Added and tested - all good there :+1:

**Review by Hugoo — APPROVED** [review]

**Pull request merged**